lib/SparkPost/APIResource.php

```diff
@@ -177,14 +177,15 @@ private function buildBody($options) {
    * @return array Result set of action performed on resource
    * @throws APIResponseException
    */
-  private function callResource( $action, $resourcePath=null, $options=[] ) {
-    $action = strtoupper($action); // normalize
+  public function callResource( $action, $resourcePath=null, $options=[] ) {
```

Comment:
this one may not be public. try `protected` instead, than you can still access it